### DISCUSSION TITLE
Readme refers to non-existent section on trash tags

### DISCUSSION THREAD

**Discussion by jayrajput** [ahyatt/ekg#51] [Q&A]
> There’s a section on these trash tags below which goes into more detail

There is no such section for trash tags referenced in prefix tags.

https://github.com/ahyatt/ekg/blob/615dc05f2aa06d0f65aed57fb3996a5cd22af867/README.org?plain=1#L84

There is a section in the docs.org though

https://github.com/ahyatt/ekg/blob/main/doc/ekg.org#9-the-trash

**Comment by ahyatt** [comment]
I've fixed this, by removing that whole part, in commit 05c3fa2cb09db7529ded496d733d553a8302a60b.  Thank you for pointing this out!  For things like this that are clearly problems, please open an issue instead, which is better for tracking purposes.

**Reply by jayrajput** [reply]
Sure. I will do that going forward.
